src/pickers/TaskExecutorsPicker.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import {
+  useTranslations, Autocomplete, useModulesManager, useGraphqlQuery,
+} from '@openimis/fe-core';
+
+function TaskExecutorsPicker({
+  multiple = true,
+  withLabel,
+  onChange,
+  value,
+  readOnly,
+  required,
+}) {
+  const [searchString, setSearchString] = useState();
+  const modulesManager = useModulesManager();
+  const { formatMessage } = useTranslations('tasksManagement', modulesManager);
+
+  const { isLoading, data, error } = useGraphqlQuery(
+    `
+    query TaskExecutorsPicker($str: String) {
+      users(str: $str) {
```

Comment:
How str filter for user works? Is it icontains filter on login name, mail, first and last name?

Reply:
Actually in this case it's not working at all. I forgot to remove it. I'm using filterOptions function to filter users - this allows to filter them without sending redundant queries.